src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Internal/ModelBindingHelper.cs

```diff
@@ -450,13 +450,13 @@ public static class ModelBindingHelper
 
             await binder.BindModelAsync(modelBindingContext);
             var modelBindingResult = modelBindingContext.Result;
-            if (modelBindingResult != null && modelBindingResult.Value.IsModelSet)
+            if (modelBindingResult != null && modelBindingResult.IsModelSet)
```

Comment:
:+1: - will do a find all references for `.Result` in the product and double check that I didn't miss any.